parsers/to-flutter/to-flutter.spec.ts

```diff
@@ -36,7 +35,7 @@ describe('To Flutter', () => {
           measurement: {
             fileName: 'custom-measurements-file-name.dart',
             className: 'CustomTheme',
-            devicePixelRatio: 3,
+            devicePixelRatio: 3.0,
```

Comment:
Why add `3.0` instead of `3`?

Reply:
It was initially `3.0`. We can set it back to 3.